parastell.py

```diff
@@ -184,17 +195,19 @@ def native_export():
         cubit.cmd("mesh surface all")
 
         # Export DAGMC file
-        cubit.cmd(f'export cf_dagmc "{cwd + "/dagmc.h5m"}" overwrite')
+        export_path = Path(dir) / f'{filename}.h5m'
+        cubit.cmd(
+            f'export cf_dagmc "{export_path}" overwrite'
+        )
 
-    # Get current working directory
-    cwd = os.getcwd()
+    dir = export['dir']
+    filename = export['h5m_filename']
```

Comment:
make these `Path` objects from the beginning?

Reply:
Regarding `filename`, since we combine it with the string `.h5m`, the syntax for compiling the whole export path would stay the same even if we convert it to a `Path` object. For example,
```
export_path = dir / f'{filename}.h5m'
```
where `dir` and `filename` are `Path` objects per your suggestion. In this case, I think it would make the most sense to keep `filename` as a string. However, if we have users supply `export['h5m_filename]` with the .h5m extension, e.g.,
```
export = {
    ...
    'h5m_filename': 'dagmc.h5m',
    ...
}
```
then converting `filename` to a `Path` object from the outset would make sense. What are your thoughts?

Reply:
or
```
export_path = dir / filename.with_suffix('.h5m')
```